dart/lib/src/protocol/sentry_event.dart

```diff
@@ -262,12 +263,15 @@ class SentryEvent {
       json['exception'] = {
         'values': [exception.toJson()].toList(growable: false)
       };
-    } else if (stackTrace is SentryStackTrace) {
+    }
+
+    if (stackTrace != null) {
```

Comment:
it was an `else if` and it was correct, not sure why you have changed it

Reply:
✅